test/TEST-73-LOGIND/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -e
+
+TEST_DESCRIPTION="Test various logind features"
+IMAGE_NAME="logind"
+
+# shellcheck source=test/test-functions
+. "${TEST_BASE_DIR:?}/test-functions"
+
+test_append_files() {
+    image_install -o useradd userdel passwd crond crontab pkill
```

Comment:
Now `userdadd`, `userdel`, and `pkill` are included in the default image.
I do not think it is necessary to build new image just for `crond` and `crontab` (and `passwd` if it is really necessary).
How about to optionally install them in `install_basic_tools()`?

Reply:
Thanks for the suggestion!